cmd/producer/main.go

```diff
@@ -56,14 +56,18 @@ func main() {
 		e := scanEvent(scanner)
 		log.Info(ctx, "sending hello-called event", log.Data{"helloCalledEvent": e})
 
-		bytes, err := schema.ContentPublishedEvent.Marshal(e)
```

Comment:
not sure you need the err2 - you could either name it something more sensible like `kafkaErr` or just reuse the existing `err`

Reply:
This is now addressed.